app/src/main/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListPresenter.java

```diff
@@ -2,32 +2,40 @@
 
 import android.support.annotation.NonNull;
 
+import java.util.ArrayList;
 import java.util.List;
 
 import io.intrepid.contest.base.BasePresenter;
 import io.intrepid.contest.base.PresenterConfiguration;
 import io.intrepid.contest.models.Category;
 import io.intrepid.contest.models.Contest;
 
-
-class CategoriesListPresenter extends BasePresenter<CategoriesListContract.View> implements CategoriesListContract.Presenter {
-
+class CategoriesListPresenter extends BasePresenter<CategoriesListContract.View> implements CategoriesListContract.Presenter, CategoryClickListener {
     private final Contest.Builder contestBuilder;
+    private final String categoryName = "Score";
```

Comment:
Do you need to create this default data in your Presenter?  Maybe a new Contest should be initialized with a default Category when it first gets created and then this Presenter wouldn't even have to worry about it, it just displays what's there.